be1-go/hub/election.go

```diff
@@ -157,6 +161,20 @@ func (c *electionChannel) castVoteHelper(publish message.Publish) error {
 		}
 	}
 
+	senderPK := msg.Sender.String()
```

Comment:
In #370 I updated the encoding of the attendees' key that are stored on the server. The public keys are store in Base64URL in that PR. So we will need to change this accordingly later.

Reply:
Yes exactly